components/gitpod-db/src/typeorm/workspace-cluster-db-impl.ts

```diff
@@ -58,7 +58,7 @@ export class WorkspaceClusterDBImpl implements WorkspaceClusterDB {
         let qb = repo
             .createQueryBuilder("wsc")
             .select(Object.keys(prototype).map((k) => `wsc.${k}`))
-            .where("TRUE = TRUE"); // make sure andWhere works
```

Comment:
Was this really in prod?..

Reply:
Yes. I guess because the rest of this method conditionally adds a bunch of `andWhere` clauses:

https://github.com/gitpod-io/gitpod/blob/f364bdc8c76741c51b05ead716cae78e03935ec8/components/gitpod-db/src/typeorm/workspace-cluster-db-impl.ts#L62-L79

so the `where("TRUE=TRUE")` ensures that appending with `andWhere` later will always work.

Reply:
> Was this really in prod?..

Jep. Feels like an ok trade-off for the benefit it gives in readability. :upside_down_face: 

FYI: The TypeORM version we're using is now able to handle one or more `andWhere` _without_ a leading `where`, so it's not necessary anymore, anyway. :lotus_position: